config/routes.rb

```diff
@@ -443,6 +443,8 @@
 
     get '/sign-in' => 'sessions#new'
     get '/sign-out' => 'sessions#destroy'
+    post '/sign-in-by-email' => 'sessions#sign_in_by_email', as: :sign_in_by_email
```

Comment:
Are these correct? The magic link email points users to `GET /provider/sign-in-by-email?token=325AB....`

Reply:
Review app gives me a 404